setup.py

```diff
@@ -18,6 +18,7 @@
     "diskcache",
     "termcolor",
     "flaml",
+    "PyPDF2",
```

Comment:
It would be better to put this in the extras_require of `retrievechat`

Reply:
Done 👍